src/hotspot/cpu/aarch64/immediate_aarch64.cpp

```diff
@@ -124,9 +124,22 @@ static inline uint32_t uimm(uint32_t val, int hi, int lo)
   return pickbits32(val, hi, lo);
 }
 
-// SPEC bits(M*N) Replicate(bits(M) x, integer N);
-// this is just an educated guess
-
+// SPEC
+//
+// bits(M*N) Replicate(bits(M) x, integer N);
```

Comment:
```suggestion
// bits(M*N) Replicate(bits(M) B, integer N);
```
...because the next text references "bit string B"?

Reply:
Indeed.